Mathlib/Data/Analysis/Filter.lean

```diff
@@ -266,9 +266,9 @@ protected def inf {f g : Filter α} (F : f.Realizer) (G : g.Realizer) : (f ⊓ g
       pt := (F.F.pt, G.F.pt)
       inf := fun ⟨a, a'⟩ ⟨b, b'⟩ ↦ (F.F.inf a b, G.F.inf a' b')
       inf_le_left := fun _ _ ↦ inter_subset_inter (F.F.inf_le_left _ _) (G.F.inf_le_left _ _)
-      inf_le_right := fun _ _ ↦ inter_subset_inter (F.F.inf_le_right _ _) (G.F.inf_le_right _ _) },
-    by
-    cases F ; cases G ; substs f g ; simp only [CFilter.toFilter, Prod.exists] ; ext
+      inf_le_right := fun _ _ ↦
+        inter_subset_inter (F.F.inf_le_right _ _) (G.F.inf_le_right _ _) }, by
```

Comment:
This is another misleading `by`. In general you don't want to use hanging `by` in the second/later argument of a tuple or anonymous structure constructor. There are several alternative ways to reformat it, but assuming only whitespace changes I would say to just leave the `by` on its line (and indent the subsequent block).